tests/conftest.py

```diff
@@ -111,8 +124,23 @@ def wait_until_status(function, instance_id, status, max_retries=10, sleep_time=
         iteration += 1
         sleep(sleep_time)
         if is_instance_of_report:
-            instance = function(instance_id).actual_instance
+            instance = function(*params).actual_instance
         else:
-            instance = function(instance_id)
+            instance = function(*params)
 
     return instance
+
+
+def repeat_request_until_http_code_changes(
+    function, params, max_retries=10, sleep_time=1
+):
+    iteration = 0
+    while iteration <= max_retries:
+        try:
+            instance = function(*params)
+            break
+        except onfido.ApiException:
+            sleep(sleep_time)
+            iteration += 1
+            print(iteration)
```

Comment:
Peraphs this `print` has been forgotten?

Reply:
Yes, removed. Thanks